Thanks @bransgithub!

FYI there is at least one test that might need updating:

```
=================================== FAILURES ===================================
___________________ ChannelOrderTestCase.test_channel_order ____________________
self = <kolibri.core.content.test.test_data_migrations.ChannelOrderTestCase testMethod=test_channel_order>
    def test_channel_order(self):
        ChannelMetadata = self.apps.get_model("content", "ChannelMetadata")
        pos = 1
        for channel in ChannelMetadata.objects.all():
>           self.assertEqual(channel.order, pos)
E           AssertionError: 0 != 1
kolibri/core/content/test/test_data_migrations.py:114: AssertionError
```

For info on testing, see https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#automated-testing

--- a/kolibri/core/content/utils/annotation.py
+++ b/kolibri/core/content/utils/annotation.py
@@ -478,6 +478,8 @@ def calculate_next_order(channel, model=ChannelMetadata):
     latest_order = model.objects.latest("order").order
     if latest_order is None:
         channel.order = 1
-    else:
+
+    if channel.order is None or channel.order == 0:
         channel.order = latest_order + 1
+
     channel.save()

